tests/text/test_tabstops.py

```diff
@@ -47,9 +47,31 @@ def it_raises_on_indexed_access_when_empty(self):
         with pytest.raises(IndexError):
             tab_stops[0]
 
+    def it_can_add_a_tab_stop(self, add_tab_fixture):
+        tab_stops, new_pos, new_len, expected_xml = add_tab_fixture
+        tab = tab_stops.add_tab(new_pos)
+        assert len(tab_stops) == new_len
+        assert isinstance(tab, TabStop)
+        # assert tab_stops.???? == expected_xml
+
```

Comment:
The xml comparison is failing since the TabStop doesn't yet have a position property so the correct xml cannot be generated.  Do I remove position references from the expected_xml for now or do I need to implement tab_stop.position at this stage to be able to pass the unit test?